web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/SelectSecurityGroups.tsx

```diff
@@ -249,7 +250,7 @@ function allowsOutbound(sg: SecurityGroup): boolean {
 function allowsOutboundToPorts(rule: SecurityGroupRule): boolean {
   const publicECRPort = 443;
   const proxyPort = window.location.port;
-  if (!proxyPort || proxyPort == '443') {
+  if (!proxyPort || proxyPort === '443') {
```

Comment:
I don't know if this would actually be a problem, but the rule of thumb seems to be use `===` if you want JS to do the sane thing lol

Reply:
yeah we most of the time intend strict equal (===) b/c it matches both type and value. this `2  == '2'` is true, but `2  === '2'` is false (b/c number != string)